demos/k8s-demo/01_create_db.sh

```diff
@@ -18,13 +18,20 @@ echo Ready!
 # create database
 echo ">>--- Create database"
 
-kubectl apply -f etc/pg.yml
+kubectl --namespace quick-start-db \
+  apply -f etc/pg.yml
 
 # Wait for it
 wait_for_app quick-start-backend quick-start-db
 sleep 3
 
 kubectl --namespace quick-start-db \
-    exec -it $(get_first_pod_for_app quick-start-backend quick-start-db) -- psql -U ${DB_ADMIN_USER} -c "
+  exec \
+  -it \
+  $(get_first_pod_for_app quick-start-backend quick-start-db) \
+  -- \
+  psql \
+  -U ${DB_ADMIN_USER} \
+  -c "
```

Comment:
This could use better indentation otherwise it looks like it's all just params to kubectl exec

Reply:
addressed